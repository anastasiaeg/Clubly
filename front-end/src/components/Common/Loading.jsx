import React from 'react';
import { HashLoader } from 'react-spinners';
import './common.css';

export default class Loading extends React.Component {
    render() {
        return (
            <div className="flex-center">
                <HashLoader />
            </div>
        );
    }
}