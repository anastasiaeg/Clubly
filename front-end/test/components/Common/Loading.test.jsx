import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../../../src/components/Common/Loading';

describe('Loading', () => {
    test('should render', () => {
        const component = renderer.create( <
                Loading / >
            )
            .toJSON();
        expect(component).toMatchSnapshot();
    });
})