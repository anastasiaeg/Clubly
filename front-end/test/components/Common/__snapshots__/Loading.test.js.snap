// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render 1`] = `
<div
  className="flex-center"
>
  <div
    className="css-1o3m8w0"
  >
    <div
      className="css-1l4m1kc"
    />
    <div
      className="css-50w1hy"
    />
  </div>
</div>
`;
